notify/impl.go

```diff
@@ -957,3 +957,10 @@ func (a *loginAuth) Next(fromServer []byte, more bool) ([]byte, error) {
 	}
 	return nil, nil
 }
+
+func truncate(s string, l int) string {
```

Comment:
You requested it here https://github.com/prometheus/alertmanager/pull/295#discussion_r58917697

But I don't think we actually have it anywhere as this PR exists as the other one went stale.